packages/tailwindcss/src/index.ts

```diff
@@ -92,7 +108,7 @@ async function parseCss(
         throw new Error('`@source` cannot have a body.')
       }
 
-      if (parent !== null) {
+      if (parent !== null && parent.kind !== 'context') {
```

Comment:
It might be overly complicated but it would be neat if `context` nodes were completely transparent in walk. This would mean that parent pointed to the closest non-context node. It would eliminate all of these checks.

I don't know if its worth it but something to think about.

Reply:
Yea I think this makes sense, especially since it's just a one liner implementation!